contracts/main/proxy-wallet/ProxyWallet.sol

```diff
@@ -3,33 +3,23 @@ pragma solidity 0.8.17;
 
 import "./FathomAuth.sol";
 import "./FathomNote.sol";
-import "./ProxyWalletCache.sol";
+import "./ProxyActionsStorage.sol";
 
 /// @dev Allows code execution using a persistant identity This can be very useful to execute a sequence of atomic actions. Since the owner of
 // the proxy can be changed, this allows for dynamic ownership models i.e. a multisig
 contract ProxyWallet is FathomAuth, FathomNote {
-    ProxyWalletCache public cache; // global cache for contracts
```

Comment:
ah, okei. proxyCache's read and write fn not really needed since we will hardcode proxyActions lib to limit proxyWallet's behavior.